### PR TITLE
Classify ZyncWorker::UnprocessableEntityError as Bugsnag INFO level

### PR DESCRIPTION
**Pull request by guicassolato**

### CODE REVIEW

File: app/workers/zync_worker.rb [hunk 1]
```diff
@@ -54,6 +54,8 @@ def find_model(name)
 
   class UnprocessableEntityError < InvalidResponseError; end
 
+  Bugsnag::Middleware::ClassifyError::INFO_CLASSES << UnprocessableEntityError.to_s
```
Comment (hallelujah) [inline app/workers/zync_worker.rb]: Hmm can't we do that in a config file?

Comment (guicassolato) [reply app/workers/zync_worker.rb]: Sorry, I was just following what we have in the file: https://github.com/3scale/porta/blob/795ec0d97ee5893da92e1744440227bb8fb953cf/app/workers/zync_worker.rb#L192

Comment (hallelujah) [reply app/workers/zync_worker.rb]: See https://docs.bugsnag.com/platforms/ruby/sidekiq/customizing-error-reports/#the-report-object

**Comment by guicassolato** [comment]
We are merging this now, but better think about a more sophisticated way to configure report severities for classes in general.

**Pull request merged**